packages/@biomejs/biome/README.md

```diff
@@ -105,13 +108,9 @@ Biome offers a simple sponsorship program that allows companies to get visibilit
 
 We use [Polar.sh](https://polar.sh/biomejs) to up-vote and promote specific features that you would like to see and implement. Check our backlog and help us:
 
-<a href="https://polar.sh/biomejs"><img src="https://polar.sh/embed/fund-our-backlog.svg?org=biomejs" /></a>
```

Comment:
Why not keep both?

Reply:
I thought it wasn't very clear since you raised the question. If it was unclear to you, it might be for other people.

Reply:
What do you mean? What is not clear?

Reply:
I'm referring to the discussion we had in discord